.editorconfig

```diff
@@ -0,0 +1,18 @@
+# https://editorconfig.org/
+
+root = true
+
+[*]
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+
+[*.py]
+indent_style = space
+indent_size = 4
+charset = utf-8
+max_line_length = 119
```

Comment:
I thought you wanted to put the line limit on the 100th character here, right?

Reply:
Ah, yes :+1: Copy-paste mistake from a previous file. I updated it.